cosmo_tester/framework/test_hosts.py

```diff
@@ -736,6 +746,38 @@ def _update_instances_list(self, outputs):
                     self._tmpdir
                     )
 
+    def _save_manager_logs(self, err_raised, purge=True):
+        self._logger.debug('_save_manager_logs started')
+        logs_dir = os.environ.get('CFY_LOGS_PATH_LOCAL')
+        test_path = self._tmpdir.name
+        if not logs_dir:
+            self._logger.debug('not saving manager logs')
+            return
+        if os.environ.get('SKIP_LOG_SAVE_ON_SUCCESS') and err_raised:
```

Comment:
I think it's enough to print this msg once when the flag is up, because you can not do anything with different log lines about not saving

Reply:
I removed the log output for when skipping saving on successful tests runs. I hope I understood what you meant.